app/components/Header/Menu/MenuNav.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import NavLink from "./NavLink";
+
+export type MenuType = {
+  text: string;
+  url: string;
+};
+
+export type MenutypeMain = {
+  links: Array<MenuType>;
+};
+
```

Comment:
You can define a single type for menu and kindly add it to its own file i.e Menu.types.ts